components/brave_rewards/browser/rewards_p3a.cc

```diff
@@ -45,7 +45,7 @@ void RecordWalletState(const WalletState& state) {
       answer = 1;
     }
   }
-  UMA_HISTOGRAM_EXACT_LINEAR("Brave.Rewards.WalletState", answer, 5);
+  UMA_HISTOGRAM_EXACT_LINEAR("Brave.Rewards.WalletState", answer, 6);
```

Comment:
You bump a lot of these bins, but don't mention them in the commit message.

If I understand the comment on the `UMA_HISTOGRAM_EXACT_LINEAR` correctly, values at the third `exclusive_max` argument are actually recorded and reported, it's just that they're lumped in with any out-of-range values. Since we only record the few discrete values here, there's no functional change. Is that correct?

Reply:
Yes, that's right. From my testing it seems that values that exceed the exclusive max are still reported. I just updated these values for good practice.

Reply:
Ok, great. That would be a good thing to add to the body of the commit message.

Reply:
I'd actually advice against this change, e.g. changing bins for P2A would break the perturbation code where we count the number of buckets. Actually that `exclusive_max` thing is something new, earlier it was `value_max` that was less confusing for our purposes :)

Reply:
Thanks for informing me about this @iefremov , was not aware of this functionality. I just looked at the pertubation code and it seems to apply to P2A metrics only. It also seems that histogram bucket_count is only used for the pertubation function. If you think that this change will affect P3A reporting, please let me know and I'll revert this immediately. :)

Reply:
Currently the pertubation is only applied to `Brave.P2A.*` metrics, but it's a general differential-privacy technique. We want to leave room to apply it to other metrics in the future.